### DISCUSSION TITLE
Default options not appearing in lookup data?

### DISCUSSION THREAD

**Discussion by akapsch** [sheltermanager/asm3#1524] [Q&A / Help]
I setup an ASM server to test with and to create a demo for our team using our data so they can get a better idea of what ASM is capable of. I went to start configuring things, and I noticed something odd. When I went to the lookup data section and selected person flags, the list appeared blank, as I would expect for a new server. I populated a few values. When I went to add a person and give them flags, a whole list of flags appeared that aren't in the lookup data setup. Is this expected? Where are these values coming from?

Here is what ASM shows for my person flags:
![PersonFlags1](https://github.com/user-attachments/assets/09c74f61-e7dc-44d7-936f-8c7af84b4aa6)

Here is what I see when I go to add a person:
![AddPerson](https://github.com/user-attachments/assets/1b39c9ce-2fb6-4b71-ab05-304c54b3c9be)

**Comment by bobintetley** [comment]
They are the built in flags. Since most of the system's functionality relies on them in some way, they can't be removed.

**Reply by akapsch** [reply]
Makes sense, but it isn't very intuitive. It would be good to display them, but make them greyed out or similar to show the user that they exist but can't be modified or deleted. Otherwise if you didn't know they existed, you would end up recreating them and having duplicates as shown in my example. Perhaps putting in a check when creating new flags to verify there isn't already an existing flag would be good too.

**Comment by bobintetley** [comment]
Thanks for the feedback. I'll put this on our dev list to do.

**Reply by Pletoo** [reply]
Similarly, the built in animal flags are not displayed in their lookup list.
Nor are they on the bulk change animal screen.
